lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
```

Comment:
This doesn't mutate feature_collection, or even use it, actually. Instead of passing it through like this, we should restructure the code around it. See my comments for `import_admins`.